bin/templates/gradle.properties

```diff
@@ -0,0 +1,4 @@
+org.gradle.daemon=true
+org.gradle.jvmargs=-Xmx2048m
+android.useAndroidX=false
+android.enableJetifier=false
```

Comment:
Why is this file being added here?

Reply:
this is an unintended artefact of the new unit test. I will remove it

Reply:
the new unit test generates this file every time it runs

Reply:
You may be able to use spies to prevent actual code logic from running. I fear if unit tests causes side effects such as creating files, then they may not run in a consistent manner if the file already exists.